performance-tests/crypto-sidecar/index.js

```diff
@@ -0,0 +1,53 @@
+const Koa = require("koa");
+const crypto = require("@nmshd/crypto");
+const koaBody = require("koa-body").koaBody;
+
+const app = new Koa();
+
+const port = 3000;
+
+app.use(
+    koaBody({
+        jsonLimit: "1kb"
+    })
+);
+
+app.use(async (ctx, next) => {
+    const start = Date.now();
+    await next();
+    const ms = Date.now() - start;
+    console.log(`${ctx.method} ${ctx.url} - ${ms} ms`);
+});
+
+app.use(async function (ctx, next) {
+    if (ctx.method !== "GET" || ctx.path !== "/keypair") return await next();
+
+    ctx.body = JSON.stringify(await crypto.CryptoSignatures.generateKeypair());
+    ctx.status = 200;
+});
+
+app.use(async function (ctx, next) {
+    if (ctx.method !== "GET" || ctx.path !== "/password") return await next();
```

Comment:
I don't know koa, but is there no `app.get("/password", callback)`?

Reply:
I only found out about [koa-router](https://www.npmjs.com/package/koa-router) when finishing this up. It's something we can add if deemed necessary.

Reply:
If koa doesn't support this out of the box I would really vote for removing koa and adding express.

Reply:
I agree with @jkoenig134. The current code looks really ugly.